SCF-Session-7/sparse/SparseMatrix.java

```diff
@@ -0,0 +1,305 @@
+package sparse;
+/**
```

Comment:
**Need to re-submit the whole assignment because it is completely deigned with a fixed hard coded matrix size.**